### ISSUE TITLE
"All fields" select may be more accessible as dropdown

### ISSUE THREAD

**Issue by jvine** [sul-dlss/SearchWorks#375]
Question for @jkeck @mejackreed 

In J Sutton's Blacklight review, she commented that the behaviour of the "all fields" select in the search form was different from the dropdowns elsewhere in the page (like "per page"), and that it would be better to be consistent, preferring the dropdown behaviour.

The reason for the difference is that the select is a form field that is submitted with the form, and technically doesn't cause anything to happen on its own (as a menu option in a dropdown would do).

However, as we add field-specific typeahead behaviour to the search field, the select technically WILL cause something to happen - i.e. it will change the behaviour of the typeahead. And as a cosmetic issue, it is much easier to style a dropdown than a select.

So the question is:  is it possible to replace the current select with a dropdown, without breaking Blacklight behaviour?
